From f9a939b036881eb6ecddab42dbca436240d5c6ea Mon Sep 17 00:00:00 2001
From: Justin Mitchell <jumitche@redhat.com>
Date: Thu, 2 Nov 2017 10:50:14 -0400
Subject: [PATCH] nfs-utils: Remove dependency on libnfsidmap-devel package

With commit a867b45 (Release: 2.2.1) the libnfsidmap-devel
package is now build from with the nfs-utils source tree.
So the programs dependent on libnfsidmap-devel need to
look inward verses outward for the nfsidmap.h header file.

Signed-off-by: Steve Dickson <steved@redhat.com>
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 9b2a502..c7651f2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -441,6 +441,8 @@ if test -n "$path_plugins" ; then
 fi
 AM_CONDITIONAL(PATH_PLUGINS, test -n "$path_plugins")
 
+AC_SUBST(AM_CPPFLAGS, "$AM_CPPFLAGS -I../../support/nfsidmap")
+
 dnl Check for IPv6 support
 AC_IPV6
 
-- 
1.8.3.1

